File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/formatter/tests/ScalaDocFormatterTest.java
package org.jetbrains.plugins.scala.lang.formatter.tests;

import junit.framework.Test;
import junit.framework.TestCase;
import org.jetbrains.plugins.scala.lang.formatter.FormatterTestSuite;

public class ScalaDocFormatterTest extends TestCase {
    public static Test suite() {
        return new FormatterTestSuite("/formatter/scalaDocData/") {
        };
    }
}

File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/formatter/intellij/tests/ScalaDocFormatterTest.scala
package org.jetbrains.plugins.scala.lang.formatter.intellij.tests

import org.jetbrains.plugins.scala.lang.formatter.intellij.FormatterTestBase

import java.nio.file.Path

class ScalaDocFormatterTest extends FormatterTestBase {
  override protected def relativeTestDataPath: Path = Path.of("formatter", "scalaDocData")
}
